crates/ruff_workspace/src/configuration.rs

```diff
@@ -907,6 +915,29 @@ impl LintConfiguration {
             }
         }
 
+        let removed_selectors = removed_selectors.iter().collect::<Vec<_>>();
+        match removed_selectors.as_slice() {
+            [] => (),
+            [selection] => {
+                let (prefix, code) = selection.prefix_and_code();
+                return Err(anyhow!(
+                    "Rule `{prefix}{code}` was removed and cannot be selected."
+                ));
+            }
+            [..] => {
+                let mut message =
+                    "The following rules have been removed and cannot be selected:".to_string();
+                for selection in removed_selectors {
+                    let (prefix, code) = selection.prefix_and_code();
+                    message.push_str("\n\t- ");
```

Comment:
Nit: I prefer to use spaces for alignment in terminals because tabs can be awkwardly large in the terminal, although it has the downside that we don't respect the users configured indentation.

Reply:
```suggestion
                    message.push_str("\n    - ");
```